app/js/main.js

```diff
@@ -330,6 +330,15 @@ function removeFrameReelSelection() {
 }
 
 /**
+ * Add selected frame highlight to frame.
+ * @param {Number} id The image ID to highlight.
```

Comment:
Blank line right here above `@param` (see the JSDoc annotations in file.js for the format we'll transition to once I write down some things and get it to you).